images/ansible/roles/kubernetes/defaults/main.yml

```diff
@@ -0,0 +1,9 @@
+---
+kubernetes_version: "{{ kubernetes.version if kubernetes_version_defined else '1.9.3-00' }}"
```

Comment:
Kill all the backwards compat for `kubernetes.*` variables here and switch to simple defaults.

Default for versions should probably be an empty string or undefined, and the role should default to latest available release version if not specified.

At some point we may want to consider a way to handle alpha/beta/rc releases as well, but that isn't a priority for the MVP.